<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alarm</class>
 <widget class="QDialog" name="Alarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>StopWatch</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QTimeEdit" name="timeEdit"/>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTimeEdit" name="timeEdit_2"/>
       </item>
       <item row="2" column="0">
        <widget class="QTimeEdit" name="timeEdit_3"/>
       </item>
       <item row="3" column="0">
        <widget class="QTimeEdit" name="timeEdit_4"/>
       </item>
       <item row="4" column="0">
        <widget class="QTimeEdit" name="timeEdit_5"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
